src/commands/community/nft/query.ts

```diff
@@ -130,16 +136,44 @@ export async function composeNFTDetail(
       })
     : []
 
-  const embed = composeEmbedMessage(msg, {
+  let embed = composeEmbedMessage(msg, {
     author: [
       `${capitalizeFirst(colName)}${chainName ? ` (${chainName})` : ""}`,
       ...(colImage.length ? [colImage] : []),
     ],
     description,
     image: nftImage,
     color: rarityColors[rarity?.rarity?.toUpperCase()],
-  }).addFields(fields)
-  return justifyEmbedFields(embed, 3)
+  }).addFields(attributeFields)
+
+  embed = justifyEmbedFields(embed, 3)
+
+  const activityRes = await community.getNFTActivity({
+    collectionAddress: collection_address,
+    tokenId: token_id,
+  })
+  if (activityRes.ok) {
```

Comment:
handle luôn case not ok nha -> throw APIError, tham khảo mấy command khác cho dễ